packages/build/tests/core/tests.js

```diff
@@ -380,27 +380,37 @@ test.serial('Passes the right feature flags to zip-it-and-ship-it', async (t) =>
     flags: { featureFlags: { buildbot_nft_transpile_esm: true } },
     snapshot: false,
   })
+  await runFixture(t, 'schedule', {
+    flags: { featureFlags: { zisi_parse_isc: true } },
+    snapshot: false,
+  })
+  await runFixture(t, 'schedule', {
+    flags: { featureFlags: { this_is_a_mock_flag: true, and_another_one: true } },
+    snapshot: false,
+  })
```

Comment:
(This is more of a general question, not anything blocking this PR)

I am wondering if we should break down this test function into smaller ones? For example, using data-driven testing like [we do in zip-it-and-ship-it](https://github.com/netlify/zip-it-and-ship-it/blob/ca3d7cdc1255cac19059ddd198df0db165be386e/tests/main.js#L74)? 

Also, this test function might become a performance bottleneck because each run of `runFixture(t, 'schedule')` must be done serially, due to modifying global variables (by stubbing `zipFunctions`).

However, I am also wondering whether this might not be an issue in the near future if we generalize feature flags passing to zip-it-and-ship-it as your describe it? If we do, then we will not need to test each individual flag anymore.

What do you think?

Reply:
> However, I am also wondering whether this might not be an issue in the near future if we generalize feature flags passing to zip-it-and-ship-it as your describe it? If we do, then we will not need to test each individual flag anymore.

Precisely. I agree that the current trend of adding a `runFixture` call for each feature flag is not ideal, but this will no longer be needed if we simply pass all the flags to ZISI.

Reply:
Hopefully we can also remove the stubbing on `zip-it-and-ship-it` too.
I am actually thinking this stubbing might be harder to do once this repository switches to pure ES modules (https://github.com/netlify/build/issues/3742) (which I am planning to do next week).